content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+[2]: /developers/metrics/types/?tab=rate#metric-submission-types
+[3]: /developers/metrics/types/?tab=gauge#metric-submission-types
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the Agent-side aggregations: `count`, `sum`, `min`, `max`, and `avg`. [Learn more about HISTOGRAM metric type][1].
+
+Suppose you have a histogram metric measuring `request.Latency` for two endpoints `endpoint:X` and `endpoint:Y`  on a single host:
+
+{{< img src="account_management/billing/custom_metrics/request_latency_single_host.png" alt="request_latency_single_host" responsive="true" style="width:80%;">}}
+
+The number of unique tag value combinations submitted for a `HISTOGRAM` metric with this tagging scheme is 2, but the Agent generates [up to 5 custom metrics][1] for each of the original 2 unique tag value combinations to account [for each Agent-side aggregations enabled][2]: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 10 custom metrics for this `HISTOGRAM` metric: :
+
+| Metric Name       | Tag Values             | Aggregations                            |
+|-------------------|------------------------|-----------------------------------------|
+| `request.Latency` | `host:A`, `endpoint:X` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:A`, `endpoint:Y` | `count`, `sum`, `min`, `max`, and `avg` |
+
+[1]: /developers/metrics/types/?tab=histogram
+[2]: /developers/metrics/types/?tab=histogram#metric-submission-types
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the server-side aggregations  `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
+
+Suppose you have a distribution metric measuring `request.Latency` in which the following tag value combinations are present in your data as shown below:
+
+{{< img src="account_management/billing/custom_metrics/request_latency.png" alt="Request latency" responsive="true" style="width:80%;">}}
+
+The number of unique combinations of metric name, host, and tag values submitted for a `GAUGE` metric with [this tagging scheme is 4](#counting-custom-metrics?tab=countrategauge):
+
+| Metric Name       | Tag Values                           |
+|-------------------|--------------------------------------|
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` |
+
+Given `request.Latency` is submitted as a `DISTRIBUTION`, Datadog [stores 5 custom metrics][1] for each of the original 4 unique tag value combinations to account for server side-aggregations: `count`, `sum`, `min`, `max`, and `avg`.
```

Comment:
Since `request.Latency` is submitted as a `DISTRIBUTION`, Datadog [stores 5 custom metrics][1] for each of these original 4 unique tag value combinations to account for server side-aggregations: `count`, `sum`, `min`, `max`, and `avg`.